### PR TITLE
feat(slack): add attachments support

### PR DESCRIPTION
**Pull request by DenovVasil**
## Description

slack attachments feat
demo

https://github.com/user-attachments/assets/130079be-370f-4533-9dc7-17da85721725

## Related issues

documentation PR: https://github.com/camunda/camunda-docs/pull/5081

closes #

## Checklist

- [ ] PR has a **milestone** or the `no milestone` label.

### CODE REVIEW

**Comment by johnBgood** [comment]
# QA information
## Test Environment
- Test required in (you can check both):
  - [ ] SaaS
  - [ ] SM
- [ ] Create a new test for this feature in our [Pre Release Tests](https://modeler.ultrawombat.com/projects/017779a8-9459-4feb-8b47-eeef8b2717ef--connectors-pre-release-test) for @Szik to be able to test

## Test Scope
*Please describe the test scope, happy path, edge cases that come to your mind, and whatever you think might relevant to test*

## Test Data
*Please provide the test data, if needed (files, URLs, code snippets, FEEL expressions)*

**Review by johnBgood — APPROVED** [review]

File: connectors/slack/src/main/java/io/camunda/connector/slack/outbound/caller/FileUploader.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    try (Response directCallResp = okHttpClient.newCall(request).execute()) {
+      if (directCallResp.code() != 200) {
+        String msg = EXTERNAL_URL_CALL_EX + directCallResp.message();
+        LOGGER.warn(msg);
+        throw new RuntimeException(msg);
+      }
+    }
+  }
+
+  private List<File> completeFileUpload(
```
[the remark this answers is not in the document]
Comment (DenovVasil) [reply connectors/slack/src/main/java/io/camunda/connector/slack/outbound/caller/FileUploader.java:100]: makes sense, then we will have to add a method that will get the first (single) file. Such code will seem a little strange at first glance, but more self-descriptive

**Review by Szik — APPROVED** [review]
for backport

**Review by johnBgood — APPROVED** [review]
